src/node/setupServer.ts

```diff
@@ -15,6 +16,11 @@ import * as requestHandlerUtils from '../utils/requestHandlerUtils'
 export const setupServer = (...requestHandlers: RequestHandlersList) => {
   const interceptor = new RequestInterceptor()
 
+  if (!isNode) {
+    console.error(
+      '[MSW] Failed to execute `setupServer` in the environment that is not NodeJS (i.e. a browser). Consider using `setupWorker` instead.',
+    )
```

Comment:
As a final touch, could you please add a `return` statement within this `if` condition?
This way we will short circuit the execution logic, preventing any Node-related code to run in a browser. Right now it will print the error, but continue to run, which would result into all sorts of issues. Thanks!